src/app/app.component.scss

```diff
@@ -0,0 +1,13 @@
+@media (max-width: 479px) {
+    header{
```

Comment:
Why is `header` fixed only on small screen?
I guess it could be fixed for all viewports, as seen on boosted.orange.com: it only requires to add `fixed-top` class on nav tag (see [Navbar Orange component](https://boosted.orange.com/docs/4.3/components/navbar-orange/).

It would allow to totally remove this `scss` file :)